catchup/service.go

```diff
@@ -76,6 +81,10 @@ type Service struct {
 	parallelBlocks      uint64
 	deadlineTimeout     time.Duration
 	blockValidationPool execpool.BacklogPool
+	// SyncRound, provided externally, which the ledger must keep in cache
+	syncRoundMu  deadlock.RWMutex
+	syncRoundSet bool
```

Comment:
@Eric-Warehime Minor:  `syncRoundSet` appears to be unused.  I'm wondering if it should be used, if it can be removed, or if I've missed something?

My sense is:  It can be removed.

```suggestion
```

Reply:
Yes I removed its usage due to other feedback. Looks like I forgot to take the variable out.